### PR TITLE
added logging traits documentation. 

### PR DESCRIPTION
**Pull request by Gre8t**
The logging traits needed a documentation. This fixes issue #3143 

**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Comment by Gre8t** [comment]
@astefanutti @squakez

File: examples/traits/logging/logging.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+//  kamel run ./logging.java --trait logging.enabled=true \
+//  --trait logging.json=true --trait logging.level=info
+import org.apache.camel.Exchange;
+
+public class RestDSL extends org.apache.camel.builder.RouteBuilder {
```
Comment (squakez) [inline examples/traits/logging/logging.java:23]: The class name is called RestDSL, should be Logging. Also, use Java convention and use the uppercase for class names.

File: examples/traits/logging/loggingModeline.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// camel-k: traits=logging.enabled=true trait=logging.json=true trait=logging.level=info
+
+import org.apache.camel.Exchange;
+
+public class RestDSL extends org.apache.camel.builder.RouteBuilder {
```
Comment (squakez) [inline examples/traits/logging/loggingModeline.java:22]: The class name is called RestDSL, should be LoggingMedeline. Also, use Java convention and use the uppercase for class names.

File: examples/traits/logging/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+manual log setting example using  `logging trait`:
+     
+     $kamel run ./logging.java --trait logging.enabled=true --trait logging.json=true --trait logging.level=info
+
+The output of this result would give this:
+- Logging would be enabled, but there would be no colors for easy skimming.
+- You would need to use your own jq to pretty print and parse the json output. 
+
+**Image example**
+![json](interface/json.png)
```
Comment (squakez) [inline examples/traits/logging/README.md:38]: Since it's a log trace, better copy and past the output as a code snippet

**Review by squakez — COMMENTED** [review]
Good work, thanks.

File: examples/traits/logging/README.md (hunk 3 above)
Comment (Gre8t) [reply examples/traits/logging/README.md:38]: Ok, would do that.

File: examples/traits/logging/Logging.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+//  kamel run ./logging.java --trait logging.enabled=true \
```
Comment (squakez) [inline examples/traits/logging/Logging.java:19]: Logging uppercase

File: examples/traits/logging/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Logging Trait
+
+This type of trait tells us what is going on in a running integration pod. It does this by printing logs out to the standard output.
+
+Logs are enabled by default on an integration. Just run this example:
+
+     $kamel run logging.java
```
Comment (squakez) [inline examples/traits/logging/README.md:7]: either remove the `$` or add a space to separate it from `kamel`. Same for the rest.

File: examples/traits/logging/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+[2] {"timestamp":"2022-04-24T22:50:06.842Z","sequence":185,"loggerClassName":"org.jboss.logging.Logger","loggerName":"io.quarkus","level":"INFO","message":"Profile prod activated. ","threadName":"main","threadId":1,"mdc":{},"ndc":"","hostName":"rest-dsl-56668cc6dc-9zz7r","processName":"io.quarkus.bootstrap.runner.QuarkusEntryPoint","processId":1}
+[2] {"timestamp":"2022-04-24T22:50:06.842Z","sequence":186,"loggerClassName":"org.jboss.logging.Logger","loggerName":"io.quarkus","level":"INFO","message":"Installed features: [camel-attachments, camel-bean, camel-core, camel-direct, camel-java-joor-dsl, camel-k-core, camel-k-runtime, camel-platform-http, camel-rest, cdi, smallrye-context-propagation, vertx]","threadName":"main","threadId":1,"mdc":{},"ndc":"","hostName":"rest-dsl-56668cc6dc-9zz7r","processName":"io.quarkus.bootstrap.runner.QuarkusEntryPoint","processId":1}
+
+```
+- Logging would be enabled, and it's output would be in josn. But, there would be no colors for easy skimming.
+- You would need to use your own jq to pretty print and parse the json output. 
+## using modeline
```
Comment (squakez) [inline examples/traits/logging/README.md:46]: Uppercase Using

File: examples/traits/logging/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
+# Logging Trait
+
+This type of trait tells us what is going on in a running integration pod. It does this by printing logs out to the standard output.
+
+Logs are enabled by default on an integration. Just run this example:
+
+     $ kamel run logging.java
```
Comment (squakez) [inline examples/traits/logging/README.md:7]: Logging with L uppercase...

File: examples/traits/logging/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+     $ kamel logs logging
+
+Or you can instead get the pod name and print its logs using kubectl: 
+
+     $ kubectl get pods
+
+     $ kubecttl logs <pod name>
```
Comment (squakez) [inline examples/traits/logging/README.md:15]: Probably `kubectl`

**Review by squakez — COMMENTED** [review]
Still some change is needed

**Review by squakez — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Pull request merged**